server/src/main/java/org/opentosca/toscana/plugins/lifecycle/TransformationLifecycle.java

```diff
@@ -0,0 +1,36 @@
+package org.opentosca.toscana.plugins.lifecycle;
+
+/**
+ The basic interface to implement the TransformationLifecycle
+ */
+public interface TransformationLifecycle {
+    /**
+     This method is intended to check if the  model given in a transformation context
```

Comment:
Do not start javadoc of a method with `This method..`. Instead, use `Checks if the model...`